ipn/ipnlocal/cert.go

```diff
@@ -160,6 +179,25 @@ type certFileStore struct {
 	testRoots *x509.CertPool
 }
 
+const acmePEMName = "acme-account.key.pem"
+
+func (f certFileStore) ACMEKey() ([]byte, error) {
+	pemName := filepath.Join(f.dir, acmePEMName)
+	v, err := os.ReadFile(pemName)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, ipn.ErrStateNotExist
+		}
+		return nil, err
+	}
+	return v, nil
+}
+
+func (f certFileStore) WriteACMEKey(b []byte) error {
+	pemName := filepath.Join(f.dir, acmePEMName)
+	return os.WriteFile(pemName, b, 0600)
```

Comment:
Should we use `atomicfile` here?

Reply:
Updated all three usages of `os.WriteFile` to use `atomicfile` as its own commit. PTAL.